### ISSUE TITLE
Forwarding_rule error after migration 1.4 -> 1.6

### ISSUE THREAD

**Issue by JLabarbe** [mapserver/mapcache#168]
Hello,
Just updated Mapcache 1.4 to 1.6 and followed "MIGRATION_GUIDE" for my mapcache.xml.
I had in my "services configuration" a forwarding_rule like : 
```
<forwarding_rule name="GetStyles">
    <param name="REQUEST" type="values">GetStyles</param>
    <http><url>myurl</url></http>
</forwarding_rule>
```
Transform to
```
<forwarding_rule name="GetStyles">
    <param name="REQUEST" type="values"><value>GetStyles</value></param>
    <http><url>myurl</url></http>
</forwarding_rule>
```
Even with this modification, there is the following error : 
> failed to validate requested value for dimension (REQUEST) 

Thanks for your help

**Comment by tbonfort** [comment]
@JLabarbe please check fix provided in #170

**Comment by jbo-ads** [comment]
As @JLabarbe last activity on GitHub dates back 2017 and as provided fix looks reasonable, I am tempted to merge PR #170 and close this issue.
